spec/forms/candidate_interface/immigration_status_form_spec.rb

```diff
@@ -13,7 +13,7 @@
   describe 'validations' do
     context 'when the immigration status is nil' do
       it 'validation keeps immigration_status as `nil`' do
-        form.valid?
+        expect(form.valid?).to be false
```

Comment:
```ruby
expect(form).not_to be_valid
```

Reply:
No particular reason. I guess it just came more naturally to me.